database/sqlite/dml/build.go

```diff
@@ -114,6 +118,20 @@ SELECT count(*) as count
 FROM builds
 WHERE repo_id = ?
 AND event = ?;
+`
+	// ListRepoBuildsByEvent represents a joined query
+	// between the builds & repos table to select
+	// a build for an org with a specific event type
+	// in the database.
+	ListRepoBuildsByEvent = `
```

Comment:
I don't think we want to add this query as it was removed with:

https://github.com/go-vela/server/pull/443